common/src/main/java/urlshortener/common/repository/UserRepositoryImpl.java

```diff
@@ -98,6 +101,13 @@ public Long count() {
         return -1L;
     }
 
+    @PostConstruct
+    public void meterUsuario() {
+        User u = new User("admin", "admin", "gmail", true, new Date(2000,12,12));
+        save(u);
+        System.out.println("Metido usuario admin");
```

Comment:
Please, try to use real loggers to debug your code, for example: `private static final Logger `**name_of_logger**` = LoggerFactory.getLogger(`**actual_class**`.class)`; and then, `LOG.info("Metido usuario admin" + user.toString());`. There are many types of level loggers as `.info, .error, .debug, .warning`... They are a good practice and very useful ^^